src/passes/OptimizeInstructions.cpp

```diff
@@ -1417,8 +1417,7 @@ struct OptimizeInstructions
     }
     {
       double value;
-      if (fastMath &&
-          matches(curr, binary(Abstract::Sub, any(), fval(&value))) &&
+      if (matches(curr, binary(Abstract::Sub, any(), fval(&value))) &&
```

Comment:
Also remove `fastMath` guard from here to allow canonicalize `x - (-0.0)` ==> `x + 0.0` which is always safe

Reply:
@kripken what was the verdict here? I thought we had decided this wasn't safe.